__tests__/GithubUtils-test.js

```diff
@@ -42,39 +42,121 @@ const expectedResponse = {
-        return github.getStagingDeployCash()
-            .catch(e => expect(e).toEqual(new Error('Unable to find StagingDeployCash issue with correct data.')));
-    });
+            octokit.issues.listForRepo = jest.fn().mockResolvedValue({data: [noBodyIssue]});
+            return github.getStagingDeployCash()
+                .catch(e => expect(e).toEqual(new Error('Unable to find StagingDeployCash issue with correct data.')));
+        });
+
+        test('Test finding more than one issue', () => {
+            const octokit = new Octokit();
+            const github = new GithubUtils(octokit);
+            octokit.issues.listForRepo = jest.fn().mockResolvedValue({data: [{a: 1}, {b: 2}]});
+            return github.getStagingDeployCash()
+                .catch(e => expect(e).toEqual(new Error('Found more than one StagingDeployCash issue.')));
+        });
 
-    test('Test finding more than one issue', () => {
-        const octokit = new Octokit();
-        const github = new GithubUtils(octokit);
-        octokit.issues.listForRepo = jest.fn().mockResolvedValue({data: [{a: 1}, {b: 2}]});
-        return github.getStagingDeployCash()
-            .catch(e => expect(e).toEqual(new Error('Found more than one StagingDeployCash issue.')));
+        test('Test finding no issues', () => {
+            const octokit = new Octokit();
+            const github = new GithubUtils(octokit);
+            octokit.issues.listForRepo = jest.fn().mockResolvedValue({data: []});
+            return github.getStagingDeployCash()
+                .catch(e => expect(e).toEqual(new Error('Unable to find StagingDeployCash issue.')));
+        });
     });
 
-    test('Test finding no issues', () => {
-        const octokit = new Octokit();
-        const github = new GithubUtils(octokit);
-        octokit.issues.listForRepo = jest.fn().mockResolvedValue({data: []});
-        return github.getStagingDeployCash()
-            .catch(e => expect(e).toEqual(new Error('Unable to find StagingDeployCash issue.')));
+    describe('generateVersionComparisonURL', () => {
+        const REPO_SLUG = 'test-owner/test-repo';
+        const MOCK_TAGS = [{name: '2.2.2'},{name: '2.2.1'},{name: '2.1.0-1'},{name: '2.1.0'},{name: '2.0.0-1'},{name: '2.0.0'},{name: '1.2.2-2'},{name: '1.2.2-1'},{name: '1.2.2'},{name: '1.2.1'},{name: '1.2.0'},{name: '1.1.0'},{name: '1.0.4'},{name: '1.0.3'},{name: '1.0.2'},{name: '1.0.1'},{name: '1.0.0'},{name: '0.0.1'}];
+
+        const mockGithub = jest.fn(() => ({
+            getOctokit: () => ({
+                repos: {
+                    listTags: jest.fn(() => Promise.resolve({data: MOCK_TAGS})),
+                },
+            }),
+        }));
+
+        const octokit = mockGithub().getOctokit();
+        const githubUtils = new GithubUtils(octokit);
+
+        describe('MAJOR comparison', () => {
+            test('should return a comparison url between 2.2.2 and 1.2.2', async () => {
+                const result = await githubUtils
```

Comment:
Sadly Expensify doesn't use `async/await` in any of our Javascript code. There is a good explanation of this in our [style guide](https://github.com/Expensify/Expensify.cash/blob/master/STYLE.md#esnext-are-we-allowed-to-use-insert-new-language-feature-why-or-why-not). So you'll need to convert these to promises. Sorry 🙃